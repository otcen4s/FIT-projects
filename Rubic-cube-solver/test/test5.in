455
114
114
221 331 443 655
221 331 443 655
551 333 666 544
662
662
222