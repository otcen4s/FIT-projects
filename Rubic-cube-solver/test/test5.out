455
114
114
221 331 443 655
221 331 443 655
551 333 666 544
662
662
222

113
114
114
221 332 644 555
221 332 644 555
551 332 633 444
662
662
665

111
111
111
222 333 444 555
222 333 444 555
555 222 333 444
666
666
666

111
111
111
222 333 444 555
222 333 444 555
222 333 444 555
666
666
666

